impeller/renderer/backend/gles/context_gles.cc

```diff
@@ -58,27 +59,7 @@ ContextGLES::ContextGLES(std::unique_ptr<ProcTableGLES> gl,
         std::shared_ptr<SamplerLibraryGLES>(new SamplerLibraryGLES());
   }
 
-  // Create the device capabilities.
-  {
-    device_capabilities_ =
-        CapabilitiesBuilder()
```

Comment:
Can we get rid of CapabilitiesBuilder entirely? All backends should follow the pattern of having a custom capabilities subclass.